rogue-thi-app/data/calendar.json

```diff
@@ -9,220 +9,220 @@
+    "name": {
+      "de": "Vorlesungsbeginn und Semesterstart",
+      "en": "Start of lectures and semester"
+    },
+    "begin": "2024-03-18"
   },
   {
-    "name": "Vorlesungsfrei (Ostern)",
-    "begin": "2023-04-06",
-    "end": "2023-04-11"
+    "name": {
+      "de": "Vorlesungsfrei (Ostern)",
+      "en": "Lecture-free (Easter)"
+    },
+    "begin": "2024-03-28",
+    "end": "2024-04-02"
   },
   {
-    "name": "Prüfungsanmeldung",
-    "begin": "2023-04-25",
-    "end": "2023-05-04"
+    "name": {
+      "de": "Prüfungsanmeldung",
+      "en": "Exam registration"
+    },
+    "begin": "2024-04-23",
+    "end": "2024-05-02"
   },
   {
     "name": {
       "de": "Prüfungsabmeldung",
       "en": "Exam withdrawal"
     },
-    "begin": "2023-04-25",
-    "end": "2023-06-26"
+    "begin": "2024-04-23",
+    "end": "2024-06-26"
   },
   {
-    "name": "Vorlesungsfrei (Christi Himmelfahrt, Brückentag)",
-    "begin": "2023-05-18",
-    "end": "2023-05-19"
-  },
-  {
-    "name": "Vorlesungsfrei (Pfingsten)",
-    "begin": "2023-05-26",
-    "end": "2023-05-30"
+    "name": {
+      "de": "Vorlesungsfrei (Christi Himmelfahrt, Brückentag)",
+      "en": "Lecture-free (Ascension Day, Bridge Day)"
+    },
+    "begin": "2024-05-09",
+    "end": "2024-05-10"
   },
   {
-    "name": "Vorlesungsfrei (Fronleichnam)",
-    "begin": "2023-06-08"
+    "name": {
+      "de": "Vorlesungsfrei (Pfingsten)",
+      "en": "Lecture-free (Pentecost)"
+    },
+    "begin": "2024-05-17",
+    "end": "2024-05-21"
   },
   {
-    "name": "Notenmeldung Prädikate (ZV)",
-    "begin": "2023-06-19T09:00",
-    "hasHours": true
+    "name": {
+      "de": "Vorlesungsfrei (Fronleichnam)",
+      "en": "Lecture-free (Corpus Christi)"
+    },
+    "begin": "2024-05-30"
   },
   {
     "name": {
-      "de": "Rückmeldung zum Wintersemester 2023/2024",
-      "en": "Re-registration for winter semester 2023/2024"
+      "de": "Notenmeldung Prädikate (ZV)",
+      "en": "Grade submission predicates (ZV)"
     },
-    
-    "begin": "2023-07-02",
-    "end": "2023-08-07"
+    "begin": "2024-06-19T9:00",
+    "hasHours": true
   },
   {
     "name": {
       "de": "Erweiterter Prüfungszeitraum",
       "en": "Additional exam period"
     },
-    "begin": "2023-07-04",
-    "end": "2023-07-07"
+    "begin": "2024-07-04",
+    "end": "2024-07-10"
   },
   {
     "name": {
-      "de": "Prüfungszeitraum",
-      "en": "Exam period"
+      "de": "Ende der Vorlesungszeit",
+      "en": "End of lecture period"
     },
-    "begin": "2023-07-08",
-    "end": "2023-07-20"
+    "begin": "2024-07-10"
   },
   {
     "name": {
-      "de": "Ende der Vorlesungszeit",
-      "en": "End of lecture period"
+      "de": "Prüfungszeitraum",
+      "en": "Exam period"
     },
-    "begin": "2023-07-07"
+    "begin": "2024-07-11",
+    "end": "2024-07-23"
   },
   {
     "name": {
       "de": "Notenmeldung",
-      "en": "Recording grades"
+      "en": "Grade submission"
```

Comment:
"Official translation" would be "Recording grades" but I don't know about this one.